tests/unit/dojson/test_dojson_utils_geo.py

```diff
@@ -122,7 +122,7 @@ def test_parse_institution_address_adds_country_code():
         'city': 'Beijing',
         'country': 'China',
         'country_code': 'CN',
-        'original_address': None,
+        'original_address': [],
```

Comment:
The next few changes are fine because we still return falsy types that are going to be stripped by `strip_empty_values`.